src/components/Card/Card.stories.tsx

```diff
@@ -1,135 +1,41 @@
 import * as React from "react";
-import { action } from "@storybook/addon-actions";
-import { text, boolean, select } from "@storybook/addon-knobs";
 
-import Card from "./Card";
-import Heading from "../Heading/Heading";
-import Image from "../Image/Image";
-import Button from "../Button/Button";
+import Card, { CardImage, CardHeading, CardContent, CardActions } from "./Card";
+
 import Link from "../Link/Link";
-import Icon from "../Icons/Icon";
-import { IconRotationTypes, IconNames } from "../Icons/IconTypes";
 import { LinkTypes } from "../Link/LinkTypes";
-import { ButtonTypes } from "../Button/ButtonTypes";
 import { getCategory } from "../../utils/componentCategories";
 
 export default {
   title: getCategory("Card"),
   component: Card,
 };
 
```

Comment:
This needs to be removed or else you get the duplicate story name warning in the console:

<img width="819" alt="Screen Shot 2021-08-18 at 12 12 40 PM" src="https://user-images.githubusercontent.com/1280564/129933746-19f7bf70-5eb5-4cc4-ba80-3123f6964f0a.png">

Reply:
Done.